README.md

```diff
@@ -32,6 +32,8 @@ The following environment variables can be passed in to customize run-time funct
 - DNSMASQ_EXCEPT_INTERFACE - interfaces to exclude when providing DHCP address (default "lo")
 - HTTP_PORT - port used by http server (default 80)
 - DHCP_RANGE - dhcp range to use for provisioning (default 172.22.0.10-172.22.0.100)
+- DHCP_HOSTS - a `;` separated list of `dhcp-host` entries, e.g. known MAC addresses like `00:20:e0:3b:13:af;00:20:e0:3b:14:af` (empty by default)
```

Comment:
I believe dhcp-host can get more options per host separated by comma, maybe add a link to the dnsmasq docs or extended examples?

Reply:
Yes, good point. This is why I went with `;` as separator. Will add a link.

Reply:
Done :slightly_smiling_face: